packages/frontend/app/workers/crypto/utils/nacl.ts

```diff
@@ -16,6 +16,24 @@ export async function randomBytes(length: number) {
   return nacl.randomBytes(length);
 }
 
+export async function signMessage(
```

Comment:
these 3 methods don't need to have message in the title, since they may not always be used for the full capital M Message.